### DISCUSSION TITLE
use of a current kernel variable in another kernel as a coefficient

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#18450] [Q&A General] [answered]
I would like to use the resulting value for u1 as a coefficient in the solution of value u2 where u1 is defined in the formula d/dt(u1) = C1*u1 and u2 is defined in the formula d/dt(u2)=C2*u2+C3*u1. Is there a kernel or a function that I may apply to provide me with the desired coupling?

**Comment by GiudGiud** [comment, marked as answer]
Hello

You can couple the two equations. I believe the example for coupled force does something very similar to what you want to do.

If you want to use the "resulting" value, so solve one equation, then solve the next one, you will need to use a MultiApp. 

Guillaume

**Reply by TLWise** [reply]
Thank you. I was able to do what I wanted by using combinations of CoefReaction and CoupledForce.
